### ISSUE TITLE
False positive - VIP.ValidatedSanitizedInput

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#191]
This looks like a false positive to me:

`Detected usage of a non-validated input variable: $_SERVER`

``` php
$domain = str_replace( 'www.', '', sanitize_text_field( $_SERVER['SERVER_NAME'] ) );
```
